src/MLOps.NET.SQLite/Storage/SQLiteMetaDataStore.cs

```diff
@@ -16,11 +16,17 @@ public async Task<Guid> CreateExperimentAsync(string name)
         {
             using (var db = new LocalDbContext())
             {
-                var experiment = new Experiment(name);
-                await db.Experiments.AddAsync(experiment);
-                await db.SaveChangesAsync();
-                
-                return experiment.Id;
+	            var existingExperiment = db.Experiments.FirstOrDefault(x => x.ExperimentName == name);
+
+	            if (existingExperiment == null)
```

Comment:
Just a minor comment here about the indentation being one or two spaces too many

Reply:
Should be fixed now